packages/forms/demo/~patterns/patterns.stories.mdx

```diff
@@ -18,7 +18,9 @@ The `FileUpload` component is shown here working together with the
     args={{
       accept: { 'image/jpeg': [], 'image/png': [], 'image/gif': [] },
       multiple: true,
-      type: 'image'
+      type: 'image',
+      closeAriaLabel: 'Close',
```

Comment:
[nit] same comment here as for `filelist.stories.mdx` – probably not necessary. Only `File` needs the full suite of related controls.

Reply:
Fixed: https://github.com/zendeskgarden/react-components/pull/1509/commits/3ef82da89b72839b6415eccfdb9e458bce1e4d9d